app/api/api/serializers.py

```diff
@@ -157,6 +168,377 @@ def get_data_partner(self, obj):
         )
 
 
+class ScanReportCreateSerializerFiles(DynamicFieldsMixin, serializers.ModelSerializer):
+
+    scan_report_file = serializers.FileField(write_only=True)
+    data_dictionary_file = serializers.FileField(
+        write_only=True, required=False, allow_empty_file=True
+    )
+    dataset = serializers.CharField(required=False)
+
+    class Meta:
+        model = ScanReport
+        fields = (
```

Comment:
Are these fields necessary? 
My understanding is that this serialiser is only responsible for the files? 
So I'm thinking `modelSerializer` is not necessary if it is, and it should only focus on the 2 files. 

Improving naming - I'd say `ScanReportFilesSerializer`